packages/odyssey-react-mui/src/Radio.tsx

```diff
@@ -41,36 +47,66 @@ export type RadioProps = {
    * Callback fired when the state is changed. Provides event and checked value.
    */
   onChange?: MuiRadioProps["onChange"];
+  /**
+   * Callback fired when the blur event happens. Provides event value.
+   */
+  onBlur?: MuiFormControlLabelProps["onBlur"];
 } & Pick<FieldComponentProps, "isDisabled" | "name"> &
-  SeleniumProps;
+  AllowedProps;
 
 const Radio = ({
+  inputFocusRef,
   isChecked,
   isDisabled,
   isInvalid,
   label,
   name,
   testId,
+  translate,
   value,
   onChange: onChangeProp,
+  onBlur: onBlurProp,
 }: RadioProps) => {
+  const ref = useRef<HTMLInputElement>(null);
+  useImperativeHandle(
+    inputFocusRef,
+    () => {
+      const element = ref.current;
```

Comment:
Why create a temp variable? I think `inputRef.current?.focus()` is fine. The `.current` will always exist, it just might be run depending on when it's run.

Reply:
@KevinGhadyani-Okta the scope of my PR was only to add logic for autocomplete in `DataFilters`. I am not sure what the reason for using a temp variable here was originally, but because I am unfamiliar with this code, I am unsure how to test it. Do you think we should consider this change in a future PR since it adds scope?